docs/installation-and-deployment.md

```diff
@@ -365,3 +365,66 @@ command:
 ```
 
 This script will print the passwords for each user to the console.
+
+## Access Management
+
+### Enable social login
```

Comment:
I think that in most cases Atlas will need to be used with some existing corporate identity provider rather than a (public) social identity provider like Google or Facebook. Better to put the focus on this and explain that it's also possible to use social login as a secondary option.

Would also be good to make a link to the integrations page where you describe the federated identity functionality in more detail.